### ISSUE TITLE
cli docs on setting current stack may be confusing

### ISSUE THREAD

**Issue by susanev** [pulumi/pulumi#12722]
_this feedback was submitted by way of docs feedback_

this user was on https://www.pulumi.com/docs/reference/cli/pulumi_stack/ and indicated they could not figure out how to set the current stack. if they were looking for stack select, is there anything we can do to make this more clear or easier to find?

thoughts on what else they could have been looking for?

**Comment by abhinav** [comment]
Honestly, I'm not sure how else it could be clearer.
`stack select` is mentioned further down on that page.
We could add "run X to do Y" for a bunch of cases, but that's already covered by the list of subcommands specifying what they do.
<img width="661" alt="image" src="https://user-images.githubusercontent.com/41730/234133088-7ee7819d-cd6f-4562-802a-9e8d0f697ab8.png">

**Comment by susanev** [comment]
🤷🏻 they maybe thought the see also section was other commands rather than pulumi stack commands? but im not totally sure, hopefully you noticed that in my language when i opened it. feel free to close as no-repro if yall would like.

**Comment by susanev** [comment]
see also is a strange name for that section..

**Comment by abhinav** [comment]
> see also is a strange name for that section..

Yeah, I agree that's a weird name for that section. That's what cobra auto-generates; we can look into renaming it to "subcommands" but I'm not sure if Cobra will put only subcommands there or something else too.

We _could_ add to pick a stack do this, to list stacks do that, etc. but that'll just duplicate information that's already presented further down the page—which may be okay? What do you think?

Putting limitations of Cobra's Markdown generator aside, if you have thoughts on how the CLI reference page should look, we can definitely try to make it meet those expectations.
